recipe/meta.yaml

```diff
@@ -24,97 +22,94 @@ source:
     folder: src/gpl
 
 {% set lc, vs, ve, prefix, parbuild = "\\", "$", "", "$PREFIX", "-j $CPU_COUNT" %}  # [unix]
-{% set lc, vs, ve, prefix, parbuild = "^", "%", "%", "%LIBRARY_PREFIX%", "" %}      # [win]
 build:
-  number: 1002
-  features:
-    - blas_{{ blas_variant }}  # [not win]
-  skip: true  # [win32 or (win and vc<14)]
+  number: 0
+  skip: true  # [win32 or win]
   script: |
     mkdir build
     cd build
-
-    cmake ..                                 {{ lc }}
-      -G"{{vs}}CMAKE_GENERATOR{{ve}}"        {{ lc }}
-      -DCMAKE_PREFIX_PATH="{{ prefix }}"     {{ lc }}
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
```

Comment:
Was it intentional that this is turned off for both Intel and Generic?

Reply:
mmm dunno how that happened as locally i have the bla_vendor check... im just pushing now all the changes again that you suggested... in my local version it's:
```
      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [bla_vendor == 'Intel']
      -DUSE_SVMLIGHT=OFF                                      {{ lc }}  # [bla_vendor == 'Intel']
```

Reply:
oh no i see what you mean! 🤦‍♂

Reply:
ok i've removed the:
```
      -DUSE_SVMLIGHT=OFF                                      {{ lc }}  # [bla_vendor == 'Intel']
```
and had that for all variants.... it's a licensing issue with distributing binary version of that...